wordpress-mobile/WordPress-Android
# Notifications Settings - adding "All Your Sites" button to show short list + all sites

Fixes #4866 

With a long "Your sites" list, it's difficult for users to find the general section, so this PR moves that section to the top of the screen, followed by "Your Sites" and "Other".

To test:
1. go to My Profile tab (3rd tab)
2. tap on Notification Settings
3. you should see "Sights & Sounds" section first, as in the screenshot

![screenshot_20161203-171630](https://cloud.githubusercontent.com/assets/6597771/20861996/52eec0e8-b97d-11e6-87c1-89824361e4f1.png)

cc @daniloercoli
